packages/dd-trace/src/appsec/remote_config/capabilities.js

```diff
@@ -19,5 +19,9 @@ module.exports = {
   APM_TRACING_ENABLED: 1n << 19n,
   ASM_RASP_SQLI: 1n << 21n,
   ASM_RASP_SSRF: 1n << 23n,
-  APM_TRACING_SAMPLE_RULES: 1n << 29n
+  APM_TRACING_SAMPLE_RULES: 1n << 29n,
+  ASM_ENDPOINT_FINGERPRINT: 1n << 32n,
+  ASM_SESSION_FINGERPRINT: 1n << 33n,
```

Comment:
This is not used in the code, should we delete this line and add where it is needed?